pkg/container/kubernetes_runner.go

```diff
@@ -465,7 +467,7 @@ func NewKubernetesConfig(opt ...KubernetesRunnerConfigOptions) *KubernetesRunner
 		Provider:     "generic",
 		Namespace:    "default",
 		Repo:         "ttl.sh/melange",
-		StartTimeout: 5 * time.Minute,
+		StartTimeout: 10 * time.Minute,
```

Comment:
sneaking this in as well since ~5m seems to border too closely to gke autopilot provisioning times